lib/recaptcha.rb

```diff
@@ -65,10 +65,12 @@ def self.verify_via_api_call(response, options)
     verify_hash['remoteip'] = options[:remote_ip] if options.key?(:remote_ip)
 
     reply = api_verification(verify_hash, timeout: options[:timeout])
-    reply['success'].to_s == 'true' &&
+    success = reply['success'].to_s == 'true' &&
       hostname_valid?(reply['hostname'], options[:hostname]) &&
       action_valid?(reply['action'], options[:action]) &&
       score_above_threshold?(reply['score'], options[:minimum_score])
+
+    return success, reply
```

Comment:
`return [success, reply]`

Reply:
I find my version simpler to read, why do you suggest this more verbose approach ?

Reply:
You must return an array. But in any case, @grosser already gave us a better solution.

Reply:
`return success, reply` is exactly the same as `return [success,reply]` :) 

Such as calling `foo(a: 42)` is the same as calling `foo({a: 42})`, just a style difference / preference